### DISCUSSION TITLE
After this program work AutoUpdate?

### DISCUSSION THREAD

**Discussion by MalfurionLav** [AveYo/MediaCreationTool.bat#149] [General]
I need this function, and very important this working after this program/fix

**Comment by AveYo** [comment]
Yes?

**Comment by JoshAtticus** [comment]
I think they mean "Will my computer still autoupdate through Windows Update"

**Reply by AveYo** [reply]
We should let them clarify "this function", I'm in short supply of crystal balls ;)
